apps/ledger-live-mobile/src/families/cosmos/MemoTagInput.tsx

```diff
@@ -5,8 +5,5 @@ import type { MemoTagInputProps } from "LLM/features/MemoTag/types";
 import { GenericMemoTagInput } from "LLM/features/MemoTag/components/GenericMemoTagInput";
 
 export default (props: MemoTagInputProps<CosmosTransaction>) => (
-  <GenericMemoTagInput
-    {...props}
-    valueToTxPatch={value => tx => ({ ...tx, memo: value || undefined })}
-  />
+  <GenericMemoTagInput {...props} valueToTxPatch={value => tx => ({ ...tx, memo: value })} />
```

Comment:
I don't think this is the root cause of the issue, especially because memo can still be undefined or null looking at the types
For me the issue is probably where we use the memo and don't expect it to be undefined somehow while the type seems correct

Also looking at other `MemoTagInput` for other coins, we also use the fallback to `undefined` so I would prefer another fix if possible, especially if we cannot reproduce with or without this change

Reply:
No this is **not** the root cause you're 100% right. After more tests I managed to reproduce the bug Yesterday and it has nothing to do with the memo. I updated the ticket but I forgot to close this PR sorry.